src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -692,29 +779,55 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
     internal sealed class TupleFieldSymbol : FieldSymbol
```

Comment:
Consider making this class a `private` member of `TupleTypeSymbol`, or the class is used elsewhere, move the class to a separate file.